packages/components/app/styles/components/reveal.scss

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// REVEAL COMPONENT
+//
+
+@use "../mixins/button" as *;
+
+.hds-reveal {
+  display: inline-block; // discuss if desired
+  // outline: 1px solid limegreen;
+  // outline-offset: -1px;
+}
+
+.hds-reveal__button {
+  position: relative;
+  left: -0.9375rem;
+  margin-right: (-0.9375rem * 2); // not sure if this is a good idea (trying to account for space to left and right of button)
```

Comment:
Let me know if you have a better ides for this or feel it should be modified in some way. (Moved to parent maybe?)

Reply:
Have we thought about this always having some kind of border/visual indicator? 

If it doesn't, then you have to use negative margins and it looks bad when it's open: 
![CleanShot 2023-05-30 at 12 00 03](https://github.com/hashicorp/design-system/assets/4587451/bfa0ca9e-b4a9-422e-8a17-7153c46f4d8b)

If it did, maybe everything would stay aligned whether opened or closed? It's entirely possible everyone has had this discussion ad nauseam but I thought I'd at least ask about it.

Reply:
I agree it looks a little awkward when the borders are visible on hover or focus but this is my understanding of the desired design. This trade-off was so that the text content aligns with other content when the borders aren't visible.

@cveigt Could you confirm the desired design?

Reply:
It's definitely not ideal. In general, this is a trade-off that comes with using the tertiary button. Now, what @MelSumner outlines is a real use case, and it looks particularly bad when used on a form. How much updating the style or building a custom toggle would affect the scope?

Reply:
@KristinLBradley @didoo I was looking into this issue and I don’t see many routes we can take using the tertiary button without making significantly updating its styling and changing the current states.

Here are two options I consider could work.

<img width="1201" alt="Screen Shot 2023-05-31 at 11 26 22 PM" src="https://github.com/hashicorp/design-system/assets/2142312/7bfbce47-f645-42a2-a9ba-2f5797efe56a">

**Option A** removes the offset added to the toggle, so, it is basically the button as is. I don’t like it because it looks okay (not great) in some cases and pretty off in others, which leaves us in a similar situation.

**Option B** removes the current offset and modifies the internal padding on all sides of the button. This looks much better and doesn't seem unreasonable, IMO. If this is not feasible, **option C** would be building a custom toggle/button because this is a real use case, and I worry that we may start to get comments and feedback from both consumers and other stakeholders or maybe even users/customers that it looks like a bug or a glitch in the UI, similarly to how happened with the Button/Link.

@heatherlarsen would appreciate your thoughts on this as well.

Reply:
@KristinLBradley I looked into your suggestion

> It's not that difficult to just add a border that's always visible

This is a good idea but it would look almost like a secondary button, which is not intended, and when used near other buttons or form controls, it is a bit confusing as they look similar.

Reply:
@cveigt since it _is_ a button from a functional perspective, I wonder if the differentiation should be in the pattern guidance? like, how to use a reveal when you also have some form buttons. Just a thought.

Reply:
@cveigt Thanks for putting this side by side together! Option B feels like the right fit. That would be my vote if it's feasible from a technical perspective and doesn't add too much complexity to the project.

Reply:
Going to try Option B (remove button side padding) and see what @cveigt and others think...

Reply:
The latest changes look good, @KristinLBradley

Reply:
Thanks! Will close this thread as resolved then.